suite/runsuite.cmake

```diff
@@ -546,6 +550,30 @@ if (ARCH_IS_X86 AND UNIX AND (a64_on_x86_only OR NOT arg_automated_ci))
     " OFF ${arg_package} "")
 endif ()
 
+if (ARCH_IS_X86 AND UNIX AND cross_riscvxx_linux_only)
+# Do riscv cross-compilation if cross_riscvxx_linux_only is on.
+  set(orig_extra_ctest_args ${extra_ctest_args})
+  set(run_tests OFF)
+  set(prev_optional_cross_compile ${optional_cross_compile})
+  set(prev_run_tests ${run_tests})
+  set(ARCH_IS_X86 OFF)
+  # TODO Because of riscv can't run the build tests. So set it closed here.
+  set(build_tests "BUILD_TESTS:BOOL=OFF")
```

Comment:
But the comment just says you can't *run* the tests (hmm actually it says "can't run the build tests": please reword): if you can build them, this option should stay on as we want to test building them.  If you can't build the tests yet, please update the comment.

Reply:
"#TODO i#3544: Run build tests". Is it better?

Reply:
It'd be clearer to say

`TODO i#3544: Port tests to RISCV and build them in the workflow`

Reply:
> It'd be clearer to say
> 
> `TODO i#3544: Port tests to RISCV and build them in the workflow`
Fine.